docs/get-started/introduction.md

```diff
@@ -42,9 +42,8 @@ instance of nfd-worker is supposed to be running on each node of the cluster,
 NFD-Topology-Updater is a daemon responsible for examining allocated
 resources on a worker node to account for resources available to be allocated
 to new pod on a per-zone basis (where a zone can be a NUMA node). It then
-communicates the information to nfd-master which does the
-[NodeResourceTopology CR](#noderesourcetopology-cr) creation corresponding
-to all the nodes in the cluster. One instance of nfd-topology-updater is
+creates or updates a [NodeResourceTopology](#noderesourcetopology-cr) custom
+resource object specific to this node. One instance of nfd-topology-updater is
```

Comment:
example NRT has moved recently to https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/docs/usage/custom-resources.md#noderesourcetopology
```suggestion
creates or updates a [NodeResourceTopology](#../usage/custom-resources.md#noderesourcetopology-cr) custom
resource object specific to this node. One instance of nfd-topology-updater is
```

Reply:
Thanks for spotting this! Fixed